### DISCUSSION TITLE
Which files should or shouldn't be version controlled?

### DISCUSSION THREAD

**Discussion by jorgenpt** [fzyzcjy/flutter_rust_bridge#1764] [Q&A] [answered]
FRB v2 adds a fair amount of files to the project, and I'm curious which of these should be omitted from version control, if any:

- `lib/src/rust`
- `rust/src/*_generated*.rs`
- `rust_builder/`

I couldn't find anything in the docs to indicate that any of these should or shouldn't be versioned.

**Comment by fzyzcjy** [comment, marked as answer]
I personally version control all, s.t. it is easy to see there is no magic at all.

However, ignoring the auto-generated files are equally reasonable: You can gitignore the `*generated*` things, as well as `lib/src/rust`, which are auto generated on each `flutter_rust_bridge_codegen generate` call.

Feel free to create an issue s.t. I can track this and add a doc!

**Reply by jorgenpt** [reply]
That sounds great, thanks! I guess I'll see how much churn there is in the generated files when I start making more interesting changes to the code and make a decision then. Filed an issue, #1765.

**Reply by fzyzcjy** [reply]
You are welcome!
